tests/smoke/test_notebooks_gpu.py

```diff
@@ -20,40 +20,58 @@ def test_gpu_vm():
 @pytest.mark.gpu
 def test_ncf_smoke(notebooks):
     notebook_path = notebooks["ncf"]
-    pm.execute_notebook(notebook_path, OUTPUT_NOTEBOOK, kernel_name=KERNEL_NAME)
     pm.execute_notebook(
         notebook_path,
         OUTPUT_NOTEBOOK,
         kernel_name=KERNEL_NAME,
-        parameters=dict(TOP_K=10, MOVIELENS_DATA_SIZE="100k"),
+        parameters=dict(TOP_K=10, 
+                        MOVIELENS_DATA_SIZE="100k",
+                        EPOCHS=1,
+                        BATCH_SIZE=256),
     )
     results = pm.read_notebook(OUTPUT_NOTEBOOK).dataframe.set_index("name")["value"]
+    
+    # There is too much variability to do an approx equal, just adding top values
+    assert results["map"] < 0.05
+    assert results["ndcg"] < 0.20
+    assert results["precision"] < 0.17
+    assert results["recall"] < 0.10
+
+
+@pytest.mark.notebooks
+@pytest.mark.gpu
+@pytest.mark.skip(reason="as of now, it takes too long to do a smoke test")
+def test_ncf_deep_dive(notebooks):
+    notebook_path = notebooks["ncf_deep_dive"]
+    pm.execute_notebook(notebook_path, 
+                        OUTPUT_NOTEBOOK, 
+                        kernel_name=KERNEL_NAME,
+                        parameters=dict(TOP_K=10, 
+                                        MOVIELENS_DATA_SIZE="100k",
+                                        EPOCHS=1,
+                                        BATCH_SIZE=1024),
+                       )
 
-    assert results["map"] == pytest.approx(0.047037, TOL)
-    assert results["ndcg"] == pytest.approx(0.193496, TOL)
-    assert results["precision"] == pytest.approx(0.175504, TOL)
-    assert results["recall"] == pytest.approx(0.100301, TOL)
 
 @pytest.mark.smoke
 @pytest.mark.gpu
 def test_fastai(notebooks):
-    assert get_number_gpus() >= 1
-
     notebook_path = notebooks["fastai"]
     pm.execute_notebook(notebook_path, OUTPUT_NOTEBOOK, kernel_name=KERNEL_NAME)
     pm.execute_notebook(
         notebook_path,
         OUTPUT_NOTEBOOK,
         kernel_name=KERNEL_NAME,
-        parameters=dict(TOP_K=10, MOVIELENS_DATA_SIZE="100k"),
+        parameters=dict(TOP_K=10, MOVIELENS_DATA_SIZE="100k", EPOCHS=1),
     )
     results = pm.read_notebook(OUTPUT_NOTEBOOK).dataframe.set_index("name")["value"]
 
-    assert results["rmse"] == pytest.approx(0.912115, TOL)
-    assert results["mae"] == pytest.approx(0.723051, TOL)
-    assert results["rsquared"] == pytest.approx(0.356302, TOL)
-    assert results["exp_var"] == pytest.approx(0.357081, TOL)
-    assert results["map"] == pytest.approx(0.021485, TOL)
-    assert results["ndcg"] == pytest.approx(0.137494, TOL)
-    assert results["precision"] == pytest.approx(0.124284, TOL)
-    assert results["recall"] == pytest.approx(0.045587, TOL)
\ No newline at end of file
+    assert results["rmse"] == pytest.approx(0.959352, TOL)
```

Comment:
Similar comment as above. Is there a bug that changed these numbers, if not what else?